packages/ui-library/src/theme/typography.ts

```diff
@@ -104,7 +104,7 @@ export const buildTypographyVariant = (mode: PaletteMode): TypographyOptions =>
     fontSize: '1rem', // 16/16
     fontStyle: 'normal',
     fontWeight: 400,
-    lineHeight: 'normal', // 32/16
+    lineHeight: 2, // 32/16
```

Comment:
I believe this is true, but only when body is bold

Reply:
Oh let me check that